subxt/src/events/mod.rs

```diff
@@ -55,16 +69,32 @@
 //! returned directly. Otherwise, we'll be given a corresponding tuple of `Option`'s, with exactly
 //! one variant populated each time.
 //!
-//! ```rust
-//! let mut events = api
+//! ```no_run
+//! # use futures::StreamExt;
+//! # use subxt::{ClientBuilder, DefaultConfig, PolkadotExtrinsicParams};
+//!
+//! #[subxt::subxt(runtime_metadata_path = "../artifacts/polkadot_metadata.scale")]
+//! pub mod polkadot {}
+//!
```

Comment:
Same as above; '#' out these lines?

Reply:
Actually, I can see the point of them here, since we use the `polkadot` module in non-commented-out lines below :)